ffs/types.go

```diff
@@ -93,136 +93,108 @@ type Job struct {
 	DealErrors []DealError
 }
 
-// DefaultConfig contains a default storage configuration for an Api instance.
-type DefaultConfig struct {
+// StorageConfig contains a default storage configuration for an Api instance.
+type StorageConfig struct {
```

Comment:
Now there's just `StorageConfig`. Moved extensions that were previously on `CidConfig` to `StorageConfig`